invenio_rdm_records/resources/serializers/codemeta/schema.py

```diff
@@ -20,6 +20,7 @@ class CodemetaSchema(SchemaorgSchema):
     CodeMeta schema derives from Schema.org
     """
 
+    identifier = fields.Method("get_id")
```

Comment:
This is defined by `SchemaorgSchema`, however for Codemeta it must be added as in the `identifier` field.

See [schema](https://raw.githubusercontent.com/codemeta/codemeta/2.0/codemeta.jsonld) for reference